### PR TITLE
docs: updates three-way manual use-case

### PR DESCRIPTION
**Pull request by radhikapc**
This PR is related to https://sysdig.atlassian.net/browse/DOC-3027

### CODE REVIEW

File: use-cases/manual-org-three-way.md [hunk 1]
```diff
@@ -1,224 +1,228 @@
...
+
+    See the **account-compliance** module in the diagram given below.
+
+
+
+## Overview 
+
+In this setup, you will do the following:
+
+[three-way k8s setup](./resources/org-three-way-with-sns.png)
```
Comment (wideawakening) [inline use-cases/manual-org-three-way.md:60]: exclamation for the image is missing :)

File: use-cases/manual-org-three-way.md [hunk 2]
```diff
@@ -1,224 +1,228 @@
...
+               },
+               "Action": "sts:AssumeRole"
+           }
+       ]
+   }
+   ```
+
+   For more information, see [Creating IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html).
+
+4. Save the `ARN_SYSDIG_COMPUTE_ROLE`.
```
Comment (wideawakening) [inline use-cases/manual-org-three-way.md:126]: Firstly, i propose homogenizing the document always using these two together
SysdigComputeRole (`ARN_SYSDIG_COMPUTE_ROLE`)

think we could simplify all this part just stating that
- When running the workload (EKS or ECS flavor), it must be done using the SysdigComputeRole (`ARN_SYSDIG_COMPUTE_ROLE`)

- This role can be configured to authenticate either
  - For Kubernetes cluster, [as Kubernetes Service account level](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html). see the `serviceAccount.*` properties of the cloud-connector chart we will use in further points
  - as ECS cluster, as the [TaskRole](https://docs.aws.amazon.com/AmazonECS/latest/userguide/task-iam-roles.html)

File: use-cases/manual-org-three-way.md [hunk 3]
```diff
@@ -243,66 +247,82 @@ compute; work with the SQS and access S3 resources (this last one via assumeRole
...
+     secureAPIToken: "SYSDIG_API_TOKEN"
+   telemetryDeploymentMethod: "helm_aws_k8s_org"		# not required but would help us
+   aws:
+       region: <SQS-AWS-REGION>
+   ingestors:
+       - cloudtrail-sns-sqs:
+           queueURL:"<URL_CLOUDTRAIL_SNS_SQS>"             # step 3
+           assumeRole:"<ARN_ROLE_SYSDIG_S3_ACCESS>"        # step 4
+   ```
+
```
Comment (wideawakening) [inline use-cases/manual-org-three-way.md:285]: I would add:

For additional information on authentication options, check  `serviceAccount.*` and `aws.*` values

File: use-cases/manual-org-three-way.md [hunk 4]
```diff
@@ -1,224 +1,228 @@
...
-<!--
+This use case describes setting up Secure for Cloud for a multi-AWS accounts environment with the following:
 
-all in same region
-management account - cloudtrail (no kms for quick test)
-log archive account - s3, sns, sqs
+- AWS Management account with Organization CloudTrail
+- AWS account with Amazon Simple Notification Service (SNS) and Amazon S3 is cloud object storage
 
-0.1 Provision an S3 bucket in the selected region and allow cloudtrail access
```
[the remark this answers is not in the document]
Comment (radhikapc) [reply use-cases/manual-org-three-way.md:87]: moving to the end of the doc

File: use-cases/manual-org-three-way.md (hunk 1 above)
Comment (radhikapc) [reply use-cases/manual-org-three-way.md:60]: done

File: use-cases/manual-org-three-way.md (hunk 2 above)
Comment (radhikapc) [reply use-cases/manual-org-three-way.md:126]: where is the cloud-connector chart?

File: use-cases/manual-org-three-way.md [hunk 5]
```diff
@@ -1,82 +1,350 @@
-# Secure for Cloud - Organizational Cloudtrail+SNS with S3 in a different account
+# Multi-AWS Accounts with Organizational CloudTrail and SNS with S3
```
Comment (wideawakening) [inline use-cases/manual-org-three-way.md:1]: don't like this title much, but previous one neither 😓 

what about.... "Organizational setup -  Cloudtrail+SNS in organizational account and S3 in logs account"

Comment (radhikapc) [reply use-cases/manual-org-three-way.md:1]: better

File: use-cases/manual-org-three-way.md [hunk 6]
```diff
@@ -339,13 +436,3 @@ ExecutionRole
...
-
-
-## Testing
-
-Check within Sysdig Secure
-- Integrations > Cloud Accounts
-- Insights > Cloud Activity
-
-- [Official Docs Check Guide](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/deploy-sysdig-secure-for-cloud-on-gcp/#confirm-the-services-are-working)
-- [Forcing events](https://github.com/sysdiglabs/terraform-google-secure-for-cloud#forcing-events)
```
Comment (wideawakening) [inline use-cases/manual-org-three-way.md:351]: can we add a small section with pointers to doc on howto test?
we can put it on the `../README.md` file, and optionally leave a link on each use-case

File: use-cases/README.md [hunk 7]
```diff
@@ -1,8 +1,6 @@
-# Secure for Cloud for AWS Use-Cases
+# Secure for Cloud Use Cases for AWS Environments
```
Comment (wideawakening) [inline use-cases/README.md:1]: this index is not only aws specific, but will address that in further PRs. let's close this one

**Review by wideawakening — APPROVED** [review]

**Pull request merged**